ci-bootstrap.sh

```diff
@@ -101,3 +101,18 @@ else
   colored_text "gcc version"
   gcc --version
 fi
+
+colored_text "plist integer type check"
```

Comment:
This should not be in ci-bootstrap, as it will slowdown all the ci commands and run the check multiple times. It may also break userspace apps we build with the same scripts, as they are not restricted to integer.

Even further, `-i` argument is not supported by older BSD sed.